nnvm/include/nnvm/top/tensor.h

```diff
@@ -48,6 +48,22 @@ struct SplitParam : public dmlc::Parameter<SplitParam> {
   }
 };
 
+struct StridedSliceParam : public dmlc::Parameter<StridedSliceParam> {
+  // numpy convention, only support indices, not support list.
+  Tuple<int64_t> begin;
+  Tuple<int64_t> end;
+  Tuple<int64_t> stride;
+
+  DMLC_DECLARE_PARAMETER(StridedSliceParam) {
+    DMLC_DECLARE_FIELD(begin)
+        .describe("Indices for begin of slice");
+    DMLC_DECLARE_FIELD(end)
+        .describe("Indices for end of the slice");
+    DMLC_DECLARE_FIELD(stride)
```

Comment:
For the stride, maybe should add set_default like this:
.set_default(Tuple<int64_t>()) like MXNet did.